### ISSUE TITLE
Nummular keratitis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10080]
**Preferred term label:**
Nummular keratitis

**Synonyms**
Granular subepithelial deposits surrounded by stromal haze

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0271261

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Keratoconjunctivitis, varicella

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000912
- label: Nummular keratitis
- parents: [Keratitis (HP:0000491)](https://hpo.jax.org/app/browse/term/HP:0000491)
- pmid: PMID:34816054
- def: Nummular keratitis consists of multiple scattered small round anterior corneal superficial stromal infiltrates.
- comment: Nummular means coin-shaped.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10080
- synonyms:
